src/transformations.jl

```diff
@@ -301,6 +301,9 @@ julia> standardize(UnitRangeTransform, [0.0 -0.5 0.5; 0.0 1.0 2.0])
  0.0  0.5  1.0
 ```
 """
-function standardize(::Type{DT}, X::AbstractArray{<:Real,2}; kwargs...) where {DT<:AbstractDataTransform}
+function standardize(::Type{DT}, X::AbstractArray{<:Real,1}; kwargs...) where {DT <: AbstractDataTransform}
```

Comment:
`AbstractVecOrMat{<:Real}` should replace both methods.

The docstring should also be updated, and tests added.

Reply:
Correct. `AbstractVecOrMat{<:Real}` will do the trick in `standardize`.